source/dem/force_chains_visualization.cc

```diff
@@ -102,7 +111,7 @@ ParticlesForceChains<dim, contact_model, rolling_friction_model>::
   Vector<float> force_values(triangulation.n_active_cells());
```

Comment:
Wait why is this a Vector of float? This will be in single precision... why do that?